addons/mail/static/tests/composer/composer.test.js

```diff
@@ -1160,3 +1160,12 @@ test("composer reply-to message is restored page reload", async () => {
     await openDiscuss(channelId);
     await contains(".o-mail-Composer:contains('Replying to')");
 });
+
+test.tags("html composer");
+test("html composer: basic rendering", async () => {
+    await startServer();
+    await start({ useHtmlComposer: true });
+    await openFormView("res.partner", serverState.partnerId);
+    await click("button", { text: "Send message" });
+    await contains(".o-mail-Composer-input.odoo-editor-editable");
+});
```

Comment:
We try to reduce the number of params to start and have dedicated functions instead.
Better just use getService and call setHtmlComposer in test.

Even better if the test is first asserting the standard composer, then calling the method, and checking the new one is there.